hack/tools/go.mod

```diff
@@ -4,132 +4,194 @@ go 1.17
 
 require (
 	github.com/Masterminds/semver v1.5.0
-	github.com/golangci/golangci-lint v1.32.0
+	github.com/golangci/golangci-lint v1.48.0
```

Comment:
I had to bump this because 1.32 was not working on m1
versions newer than 1.48 have some transitive dependencies that need to be fixed